### PR TITLE
Added missing JSDocs in e2e test utils.

### PR DESCRIPTION
**Pull request by v9ai**
## Description
Added/ adjusted JSDocs in e2e test utils
This PR is related to this issue: https://github.com/WordPress/gutenberg/issues/11535

## How has this been tested?
JSDocs don't affect testing

## Types of changes
Added missing JSDocs
Adjusted existing JSDocs

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

**Comment by nicolad** [comment]
Hi, @gziolo, this is the follow up PR for:
https://github.com/WordPress/gutenberg/pull/11855

File: test/e2e/support/utils/are-pre-publish-checks-enabled.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/**
+ * Verify if publish checks are enabled.
```
Comment (gziolo) [inline test/e2e/support/utils/are-pre-publish-checks-enabled.js]: Can we use the form of 3rd person for consistency?

`Verifies ...`

File: test/e2e/support/utils/clear-local-storage.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/**
+ * Clear localStorage
```
Comment (gziolo) [inline test/e2e/support/utils/clear-local-storage.js]: Can we use the form of 3rd person for consistency and end the sentence with a full stop?

`Clears the local storage.`

File: test/e2e/support/utils/convert-block.js [hunk 3]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Convert editor's block type.
```
Comment (gziolo) [inline test/e2e/support/utils/convert-block.js]: Can we use the form of 3rd person for consistency?

`Converts ...`

File: test/e2e/support/utils/convert-block.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+ *
+ * @param {?string} name Block name
```
Comment (gziolo) [inline test/e2e/support/utils/convert-block.js]: I don't think that block name is optional here.

File: test/e2e/support/utils/disable-pre-publish-checks.js [hunk 5]
```diff
@@ -3,6 +3,9 @@
  */
 import { toggleOption } from './toggle-option';
 
+/**
+ * Disable Pre-publish checks
```
Comment (gziolo) [inline test/e2e/support/utils/disable-pre-publish-checks.js]: Can we use the form of 3rd person for consistency and end the sentence with a full stop?

File: test/e2e/support/utils/enable-pre-publish-checks.js [hunk 6]
```diff
@@ -3,6 +3,9 @@
  */
 import { toggleOption } from './toggle-option';
 
+/**
+ * Enable Pre-publish checks
```
Comment (gziolo) [inline test/e2e/support/utils/enable-pre-publish-checks.js]: Can we use the form of 3rd person for consistency and end the sentence with a full stop?

File: test/e2e/support/utils/is-wp-path.js [hunk 7]
```diff
@@ -8,6 +8,13 @@ import { URL } from 'url';
  */
 import { getUrl } from './get-url';
 
+/**
+ * Checks if current url is a WordPress path.
+ *
+ * @param {string} WPPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
```
Comment (gziolo) [inline test/e2e/support/utils/is-wp-path.js]: `query` is optional so it should be `{?string}`.

File: test/e2e/support/utils/get-url.js [hunk 8]
```diff
@@ -6,6 +6,13 @@ import { URL } from 'url';
 
 import { WP_BASE_URL } from './config';
 
+/**
+ * Creates new URL by parsing base URL, WPPath and query string.
+ *
+ * @param {string} WPPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
```
Comment (gziolo) [inline test/e2e/support/utils/get-url.js]: `query` is optional so it should be `{?string}`.

File: test/e2e/support/utils/login.js [hunk 9]
```diff
@@ -5,6 +5,12 @@
 import { WP_USERNAME, WP_PASSWORD } from './config';
 import { pressWithModifier } from './press-with-modifier';
 
+/**
+ * Perform log in with specified username and password.
+ *
+ * @param {string} username String to be used as user credential.
```
Comment (gziolo) [inline test/e2e/support/utils/login.js]: Both `username` and `password` are optional so it should be `{?string}`.

File: test/e2e/support/utils/set-viewport.js [hunk 10]
```diff
@@ -3,6 +3,11 @@
  */
 import { waitForPageDimensions } from './wait-for-page-dimensions';
 
+/**
+ * Set browser viewport to specified type.
```
Comment (gziolo) [inline test/e2e/support/utils/set-viewport.js]: Can we use the form of 3rd person for consistency?

**Review by gziolo — CHANGES_REQUESTED** [review]
Thanks for opening another PR. Your contribution in this area of Gutenberg is notable. I left a few comments to address and we should be good 👍

File: test/e2e/support/utils/are-pre-publish-checks-enabled.js (hunk 1 above)
Comment (nicolad) [reply test/e2e/support/utils/are-pre-publish-checks-enabled.js]: Adjusted, thx.

File: test/e2e/support/utils/clear-local-storage.js (hunk 2 above)
Comment (nicolad) [reply test/e2e/support/utils/clear-local-storage.js]: Updated, thank you.

File: test/e2e/support/utils/convert-block.js (hunk 3 above)
Comment (nicolad) [reply test/e2e/support/utils/convert-block.js]: Sure, thx.

File: test/e2e/support/utils/convert-block.js (hunk 4 above)
Comment (nicolad) [reply test/e2e/support/utils/convert-block.js]: Agree.

File: test/e2e/support/utils/disable-pre-publish-checks.js (hunk 5 above)
Comment (nicolad) [reply test/e2e/support/utils/disable-pre-publish-checks.js]: Sure, thx.

File: test/e2e/support/utils/enable-pre-publish-checks.js (hunk 6 above)
Comment (nicolad) [reply test/e2e/support/utils/enable-pre-publish-checks.js]: Updated, thank you.

File: test/e2e/support/utils/is-wp-path.js (hunk 7 above)
Comment (nicolad) [reply test/e2e/support/utils/is-wp-path.js]: Adjusted, thx.

File: test/e2e/support/utils/get-url.js (hunk 8 above)
Comment (nicolad) [reply test/e2e/support/utils/get-url.js]: Adjusted, thx.

File: test/e2e/support/utils/login.js (hunk 9 above)
Comment (nicolad) [reply test/e2e/support/utils/login.js]: Updated, thank you.

File: test/e2e/support/utils/set-viewport.js (hunk 10 above)
Comment (nicolad) [reply test/e2e/support/utils/set-viewport.js]: Sure, updated.

**Review by gziolo — APPROVED** [review]
Awesome work, all good now 🚢

**Pull request merged**